### PR TITLE
fix(react): add title text to all elements

### PR DESCRIPTION
**Pull request by abbeyhrt**
Closes #

{{short description}}

#### Changelog

**New**

- {{new thing}}

**Changed**

- {{change thing}}

**Removed**

- {{removed thing}}

#### Testing / Reviewing

{{ Add descriptions, steps or a checklist for how reviewers can verify this PR works or not }}

### CODE REVIEW

**Review by aledavila — APPROVED** [review]
Looks good. Did we want to keep the console logs?

**Review by dakahn — APPROVED** [review]
I think this is a good temp fix while we figure out whats happening on Firefox's end--its also consistent with our other 'dropdown' style components

File: packages/react/src/components/ListBox/ListBoxMenuItem.js [hunk 1]
```diff
@@ -10,12 +10,19 @@ import React, { useEffect, useRef, useState } from 'react';
...
 import { usePrefix } from '../../internal/usePrefix';
 
+// eslint-disable-next-line no-unused-vars
 function useIsTruncated(ref) {
   const [isTruncated, setIsTruncated] = useState(false);
 
   useEffect(() => {
     const { offsetWidth, scrollWidth } = ref.current;
+    const rect = ref.current.getBoundingClientRect;
+    console.log(rect);
```
Comment (tay1orjones) [inline packages/react/src/components/ListBox/ListBoxMenuItem.js:20]:
```suggestion
```

File: packages/react/src/components/ListBox/ListBoxMenuItem.js [hunk 2]
```diff
@@ -10,12 +10,19 @@ import React, { useEffect, useRef, useState } from 'react';
...
     setIsTruncated(offsetWidth < scrollWidth);
+    console.log(offsetWidth < scrollWidth);
+    console.log('clientWidth', ref.current.clientWidth);
+    console.log('scrollWidth', ref.current.scrollWidth);
+    console.log('offsetWidth', ref.current.offsetWidth);
```
Comment (tay1orjones) [inline packages/react/src/components/ListBox/ListBoxMenuItem.js:25]:
```suggestion
```

File: packages/react/src/components/ListBox/ListBoxMenuItem.js [hunk 3]
```diff
@@ -32,17 +39,13 @@ const ListBoxMenuItem = React.forwardRef(function ListBoxMenuItem(
 ) {
   const prefix = usePrefix();
   const ref = useRef(null);
-  const isTruncated = useIsTruncated(forwardedRef?.menuItemOptionRef || ref);
```
Comment (tay1orjones) [inline packages/react/src/components/ListBox/ListBoxMenuItem.js:35]: Should we just remove the hook altogether?

File: packages/react/src/components/ListBox/ListBoxMenuItem.js [hunk 4]
```diff
@@ -10,12 +10,19 @@ import React, { useEffect, useRef, useState } from 'react';
 import PropTypes from 'prop-types';
 import { usePrefix } from '../../internal/usePrefix';
 
+// eslint-disable-next-line no-unused-vars
 function useIsTruncated(ref) {
   const [isTruncated, setIsTruncated] = useState(false);
 
   useEffect(() => {
     const { offsetWidth, scrollWidth } = ref.current;
+    const rect = ref.current.getBoundingClientRect;
```
Comment (tay1orjones) [inline packages/react/src/components/ListBox/ListBoxMenuItem.js:19]: I think this was just used for testing maybe?

```suggestion
```

**Review by tay1orjones — COMMENTED** [review]
I'm a bit out of the loop on this one - are you thinking we disable the hook temporarily while we investigate/plan a bigger fix?

**Comment by abbeyhrt** [comment]
@tay1orjones I meant to close this! DA and I found a mozilla bug causing the lack of title to show up and we wanted to throw this up so he could show the example in the relevant bug. I should have thrown the WIP on this! Since it's a bug in firefox, I think we're going to leave it where it's at for now and I can close this

**Pull request closed without merging**